c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
Today I sometimes get an error when I want to view different document types ( f.e. here /outings/736946/fr/la-jumelle-3eme-pucelle-aisha ).
"json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)". 

I am not sure if this is caused by this change or because I did some other changes elsewhere / I rebased / worked on books API, etc... (error was still showing when I used original code changed here).